crmsh/sbd.py

```diff
@@ -212,6 +239,44 @@ def _get_sbd_device_from_config(self):
         else:
             return None
 
+    @staticmethod
+    def is_delay_start():
+        """
+        Check if SBD_DELAY_START is yes
+        """
+        res = SBDManager.get_sbd_value_from_config("SBD_DELAY_START")
+        return True if res and res == "yes" else False
```

Comment:
Technically sbd determines this based on `crm_is_true()` function and whether it's explicitly set with a delay value:

https://github.com/ClusterLabs/sbd/blob/master/src/sbd-inquisitor.c#L989-L999

https://github.com/ClusterLabs/pacemaker/blob/master/lib/common/strings.c#L414-L444

Feel free to decide whether we need to make it cover all the cases of `True` here :-)

Reply:
Changed as using `return utils.is_boolean_true(res)`
Thanks!